doc/source/serve/http-servehandle.rst

```diff
@@ -152,6 +152,60 @@ To try it out, save a code snippet in a local python file (i.e. main.py) and in
     ray start --head
     python main.py
 
+.. _serve-http-adapters:
+
+HTTP Adapters
+^^^^^^^^^^^^^
+
+Ray Serve provides a suite of adapters to convert HTTP requests to ML inputs like `numpy` arrays.
+You can just use it with :ref:`Ray AI Runtime (AIR) model wrapper<air-serve-integration>` feature
+to one click deploy pre-trained models.
+Alternatively, you can directly import them and put them into your FastAPI app.
+
+For example, we provide a simple adapter for n-dimensional array.
+
+With :ref:`model wrappers<air-serve-integration>`, you can specify it via the ``input_schema`` field.
+
+.. code-block:: python
+
+    from ray.serve.http_adapters import array_to_databatch
+    from ray.serve.model_wrappers import ModelWrapper
+    from ray import serve
+
+    serve.deployment(name="my_model")(ModelWrapper).deploy(
+        my_ray_air_predictor,
+        my_ray_air_checkpoint,
+        input_schema=array_to_databatch
+    )
+
+You can also bring the adapter to your own FastAPI app using
+`Depends <https://fastapi.tiangolo.com/tutorial/dependencies/#import-depends>`_.
+
```

Comment:
please also mention that you can check out the openapi docs to see the input format!

Reply:
mentioned below :D